_drafts/2018-08-07-draft.md

```diff
@@ -9,6 +9,8 @@ Feature story text here
 
 ## News from around the web!
 
+[Simon Game](https://github.com/Factoid/circuit_playground_examples/blob/master/simon.py) by [Adrian](https://twitter.com/wpgFactoid) in CircuitPython for Circuit Playground Express
+[Mini Sam PCB board](https://github.com/bwshockley/Mini-SAM-D51) places a SAMD51 M4 CircuitPython-compatible processor on a Lego Minifigure-sized board ([picture](https://twitter.com/bwshockley/status/1024407259298717696/photo/1)).
```

Comment:
Viewing this formatted shows there's no space between these two lines. I think there needs to be a blank space here to not have them on the same line.